src/index.html

```diff
@@ -200,6 +200,13 @@
     function takeScreenshot(element) {
       if (!url.value)
         return;
+      
+      var httpRegex = /(http(s?))\:\/\//gi;
+
+      if(!httpRegex.test(url.value)) {
+        url.value = "https://" + url.value;
```

Comment:
nit: just to be consistent, can you change this to use single quotes instead?

Reply:
Done !